modules/nf-core/wipertools/reportgather/main.nf

```diff
@@ -0,0 +1,46 @@
+process WIPERTOOLS_REPORTGATHER {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/wipertools:1.1.3--pyhdfd78af_0':
+        'biocontainers/wipertools:1.1.3--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(reports)
+
+    output:
+    tuple val(meta), path("*_gathered.report") , emit: report_out
```

Comment:
Same thing here, make the `_gathered` part of the prefix.

Reply:
corrected